### ISSUE TITLE
Error test? (Oxalis outbound)

### ISSUE THREAD

**Issue by oskaremil** [difi/oxalis#212]
I've just set up a fresh workspace after being absent from Oxalis since the 3.1 release. When I completed the configuration and prerequisites I run ``mvn clean install`` to verify it all. 

_TransmissionRequestBuilderTest_ causes an _UnknownHostException_. Any one else encountering this or have I forgot something in my setup ? 

``
java.lang.RuntimeException: Problem with SMP lookup for participant 9908:991825827 and document type urn:oasis:names:specification:ubl:schema:xsd:Invoice-2::Invoice##urn:www.cenbii.eu:transaction:biitrns010:ver2.0:extended:urn:www.peppol.eu:bis:peppol5a:ver2.0:extended:urn:www.difi.no:ehf:faktura:ver2.0::2.1 at eu.peppol.smp.SmpLookupManagerImpl.getEndpointType(SmpLookupManagerImpl.java:437) at eu.peppol.smp.SmpLookupManagerImpl.getEndpointTransmissionData(SmpLookupManagerImpl.java:163) at eu.peppol.outbound.transmission.TransmissionTestModule$2.getEndpointTransmissionData(TransmissionTestModule.java:100) at eu.peppol.outbound.transmission.TransmissionRequestBuilder.build(TransmissionRequestBuilder.java:168) at eu.peppol.outbound.transmission.TransmissionRequestBuilderTest.xmlWithNoSBDH(TransmissionRequestBuilderTest.java:97) Caused by: eu.peppol.smp.SmpSignedServiceMetaDataException: Unable to find information for participant: 9908:991825827, documentType: urn:oasis:names:specification:ubl:schema:xsd:Invoice-2::Invoice##urn:www.cenbii.eu:transaction:biitrns010:ver2.0:extended:urn:www.peppol.eu:bis:peppol5a:ver2.0:extended:urn:www.difi.no:ehf:faktura:ver2.0::2.1, at url: http://B-770c6f5843e9e302de47ae4026307076.iso6523-actorid-upis.smk.peppolcentral.org/iso6523-actorid-upis%3A%3A9908%3A991825827/services/busdox-docid-qns%3A%3Aurn%3Aoasis%3Anames%3Aspecification%3Aubl%3Aschema%3Axsd%3AInvoice-2%3A%3AInvoice%23%23urn%3Awww.cenbii.eu%3Atransaction%3Abiitrns010%3Aver2.0%3Aextended%3Aurn%3Awww.peppol.eu%3Abis%3Apeppol5a%3Aver2.0%3Aextended%3Aurn%3Awww.difi.no%3Aehf%3Afaktura%3Aver2.0%3A%3A2.1 ; Unable to connect to http://B-770c6f5843e9e302de47ae4026307076.iso6523-actorid-upis.smk.peppolcentral.org/iso6523-actorid-upis%3A%3A9908%3A991825827/services/busdox-docid-qns%3A%3Aurn%3Aoasis%3Anames%3Aspecification%3Aubl%3Aschema%3Axsd%3AInvoice-2%3A%3AInvoice%23%23urn%3Awww.cenbii.eu%3Atransaction%3Abiitrns010%3Aver2.0%3Aextended%3Aurn%3Awww.peppol.eu%3Abis%3Apeppol5a%3Aver2.0%3Aextended%3Aurn%3Awww.difi.no%3Aehf%3Afaktura%3Aver2.0%3A%3A2.1 ; B-770c6f5843e9e302de47ae4026307076.iso6523-actorid-upis.smk.peppolcentral.org at eu.peppol.smp.SmpLookupManagerImpl.fetchContentsOfSmpUrl(SmpLookupManagerImpl.java:414) at eu.peppol.smp.SmpLookupManagerImpl.getServiceMetaData(SmpLookupManagerImpl.java:294) at eu.peppol.smp.SmpLookupManagerImpl.getEndpointType(SmpLookupManagerImpl.java:432) ... 38 more Caused by: java.lang.IllegalStateException: Unable to connect to http://B-770c6f5843e9e302de47ae4026307076.iso6523-actorid-upis.smk.peppolcentral.org/iso6523-actorid-upis%3A%3A9908%3A991825827/services/busdox-docid-qns%3A%3Aurn%3Aoasis%3Anames%3Aspecification%3Aubl%3Aschema%3Axsd%3AInvoice-2%3A%3AInvoice%23%23urn%3Awww.cenbii.eu%3Atransaction%3Abiitrns010%3Aver2.0%3Aextended%3Aurn%3Awww.peppol.eu%3Abis%3Apeppol5a%3Aver2.0%3Aextended%3Aurn%3Awww.difi.no%3Aehf%3Afaktura%3Aver2.0%3A%3A2.1 ; B-770c6f5843e9e302de47ae4026307076.iso6523-actorid-upis.smk.peppolcentral.org at eu.peppol.smp.SmpContentRetrieverImpl.getUrlContent(SmpContentRetrieverImpl.java:54) at eu.peppol.smp.SmpLookupManagerImpl.fetchContentsOfSmpUrl(SmpLookupManagerImpl.java:412) ... 40 more Caused by: java.net.UnknownHostException: B-770c6f5843e9e302de47ae4026307076.iso6523-actorid-upis.smk.peppolcentral.org at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:178) at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392) at java.net.Socket.connect(Socket.java:579) at java.net.Socket.connect(Socket.java:528) at sun.net.NetworkClient.doConnect(NetworkClient.java:180) at sun.net.www.http.HttpClient.openServer(HttpClient.java:432) at sun.net.www.http.HttpClient.openServer(HttpClient.java:527) at sun.net.www.http.HttpClient.<init>(HttpClient.java:211) at sun.net.www.http.HttpClient.New(HttpClient.java:308) at sun.net.www.http.HttpClient.New(HttpClient.java:326) at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:997) at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:933) at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:851) at eu.peppol.smp.SmpContentRetrieverImpl.getUrlContent(SmpContentRetrieverImpl.java:52) ... 41 more ... Removed 34 stack frames
``

**Comment by oskaremil** [comment]
I checked at http://e-business.tools/querypeppol on the Peppol ID and the document type _is_ registered...

**Comment by oskaremil** [comment]
TL;DR

java.lang.RuntimeException: Problem with SMP lookup for participant 9908:991825827 and document type urn:oasis:names:specification:ubl:schema:xsd:Invoice-2::Invoice##urn:www.cenbii.eu:transaction:biitrns010:ver2.0:extended:urn:www.peppol.eu:bis:peppol5a:ver2.0:extended:urn:www.difi.no:ehf:faktura:ver2.0::2.1 at eu.peppol.smp.SmpLookupManagerImpl.getEndpointType(SmpLookupManagerImpl.java:437)

**Comment by Tickstar** [comment]
Hi, You seem to use the SMK instead of the SML. I'm not sure if the SMK can be expected to work.

**Comment by teedjay** [comment]
Seems like you are doing lookups using PEPPOL test infrastructure (smk.peppolcentral.org).
Switch your oxalis properties from oxalis.operation.mode=TEST to PRODUCTION and you should be fine.

**Comment by oskaremil** [comment]
Thank you ! :+1:

**Comment by ahmedinweb** [comment]
Hi jay,
Even after changing to PROUCTION, I am still getting smp lookup failure. Please advise.
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running TestSuite
Configuring TestNG with: TestNG652Configurator
2015-12-01 12:52:25,615 INFO [eu.peppol.util.GlobalConfiguration] Initialising the Oxalis global configuration ....
2015-12-01 12:52:25,619 DEBUG [eu.peppol.util.OxalisHomeDirectory] Attempting to locate home dir....
2015-12-01 12:52:25,621 INFO [eu.peppol.util.GlobalConfiguration] Oxalis home directory: C:\oxalis-master\3.1
2015-12-01 12:52:25,624 INFO [eu.peppol.util.GlobalConfiguration] oxalis.keystore = C:/oxalis-master/3.1/oxalis.jks
2015-12-01 12:52:25,624 INFO [eu.peppol.util.GlobalConfiguration] oxalis.keystore.password = XXX
2015-12-01 12:52:25,624 INFO [eu.peppol.util.GlobalConfiguration] oxalis.truststore.password = peppol
2015-12-01 12:52:25,624 INFO [eu.peppol.util.GlobalConfiguration] oxalis.inbound.message.store = c:/oxalis-message-store
2015-12-01 12:52:25,624 INFO [eu.peppol.util.GlobalConfiguration] oxalis.persistence.class.path =
2015-12-01 12:52:25,624 INFO [eu.peppol.util.GlobalConfiguration] oxalis.soap.trace = false
2015-12-01 12:52:25,624 INFO [eu.peppol.util.GlobalConfiguration] oxalis.jdbc.driver.class = oracle.jdbc.driver.OracleDriver
2015-12-01 12:52:25,624 INFO [eu.peppol.util.GlobalConfiguration] oxalis.jdbc.connection.uri = jdbc:oracle:thin:@marqadb01.elcom.com:1521:DEV
2015-12-01 12:52:25,624 INFO [eu.peppol.util.GlobalConfiguration] oxalis.jdbc.user = oxalis
2015-12-01 12:52:25,624 INFO [eu.peppol.util.GlobalConfiguration] oxalis.jdbc.password = oxalis
2015-12-01 12:52:25,625 INFO [eu.peppol.util.GlobalConfiguration] oxalis.jdbc.class.path = file:///C:/ELCOM/classpath/ojdbc6.jar
2015-12-01 12:52:25,625 INFO [eu.peppol.util.GlobalConfiguration] oxalis.jdbc.validation.query = select 1 from dual
2015-12-01 12:52:25,625 INFO [eu.peppol.util.GlobalConfiguration] oxalis.jdbc.dialect = Oracle
2015-12-01 12:52:25,625 INFO [eu.peppol.util.GlobalConfiguration] oxalis.datasource.jndi.name = jdbc/oxalisDS
2015-12-01 12:52:25,625 INFO [eu.peppol.util.GlobalConfiguration] oxalis.statistics.private.key =
2015-12-01 12:52:25,625 INFO [eu.peppol.util.GlobalConfiguration] oxalis.inbound.log.config = C:/oxalis-master/3.1/logback-oxalis-server.xml
2015-12-01 12:52:25,626 INFO [eu.peppol.util.GlobalConfiguration] oxalis.app.log.config = C:/oxalis-master/3.1/logback-oxalis-app.xml
2015-12-01 12:52:25,626 INFO [eu.peppol.util.GlobalConfiguration] oxalis.pki.version = V2
2015-12-01 12:52:25,626 INFO [eu.peppol.util.GlobalConfiguration] oxalis.operation.mode = PRODUCTION
2015-12-01 12:52:25,626 INFO [eu.peppol.util.GlobalConfiguration] oxalis.connection.timeout = 5000
2015-12-01 12:52:25,626 INFO [eu.peppol.util.GlobalConfiguration] oxalis.read.timeout = 5000
2015-12-01 12:52:25,626 INFO [eu.peppol.util.GlobalConfiguration] oxalis.sml.hostname = smk.peppolcentral.org
2015-12-01 12:52:25,627 INFO [eu.peppol.util.GlobalConfiguration] Verifying properties ....
2015-12-01 12:52:25,627 DEBUG [eu.peppol.smp.SmpLookupManagerImpl] SML hostname has been overridden: [smk.peppolcentral.org]
2015-12-01 12:52:25,627 DEBUG [eu.peppol.smp.SmpLookupManagerImpl] SML hostname: smk.peppolcentral.org
2015-12-01 12:52:26,403 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
2015-12-01 12:52:26,554 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : Invoice
2015-12-01 12:52:26,670 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : Invoice
2015-12-01 12:52:26,963 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : Invoice
2015-12-01 12:52:27,135 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : Invoice
2015-12-01 12:52:27,154 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'DespatchAdvice/1.0/Eksempel5.xml'
2015-12-01 12:52:27,357 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : DespatchAdvice
2015-12-01 12:52:27,368 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFInvoice/1.6/T14-norsk-profil05.xml'
2015-12-01 12:52:27,440 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : CreditNote
2015-12-01 12:52:27,453 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'MessageLevelResponse/1.0/Use Case2.xml'
2015-12-01 12:52:27,502 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : ApplicationResponse
2015-12-01 12:52:27,505 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFInvoice/2.0/T14-Valuta-EUR.xml'
2015-12-01 12:52:27,551 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : CreditNote
2015-12-01 12:52:27,556 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFInvoice/2.0/T10-Valuta-EUR.xml'
2015-12-01 12:52:27,599 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : Invoice
2015-12-01 12:52:27,603 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'DespatchAdvice/1.0/Eksempel1.xml'
2015-12-01 12:52:27,654 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : DespatchAdvice
2015-12-01 12:52:27,658 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'DespatchAdvice/1.0/Eksempel4.xml'
2015-12-01 12:52:27,708 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : DespatchAdvice
2015-12-01 12:52:27,713 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'DespatchAdvice/1.0/Eksempel3.xml'
2015-12-01 12:52:27,766 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : DespatchAdvice
2015-12-01 12:52:27,771 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFInvoice/1.6/T14-norsk-profil05-without-endpointid.xml'
2015-12-01 12:52:27,819 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : CreditNote
2015-12-01 12:52:27,825 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'MessageLevelResponse/1.0/Use Case3.xml'
2015-12-01 12:52:27,867 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : ApplicationResponse
2015-12-01 12:52:27,870 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFOrder/1.0/Eksempelfil EHF Ordrebekreftelse.xml'
2015-12-01 12:52:27,919 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : OrderResponse
2015-12-01 12:52:27,922 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFCatalogue/1.0/Example file EHF Catalogue Response.xml'
2015-12-01 12:52:27,961 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : ApplicationResponse
2015-12-01 12:52:27,964 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFReminder/1.6/T17-norsk-profilxy.xml'
2015-12-01 12:52:28,005 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : Reminder
2015-12-01 12:52:28,007 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFOrder/SimpleOrderResponse.xml'
2015-12-01 12:52:28,052 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : OrderResponseSimple
2015-12-01 12:52:28,055 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'DespatchAdvice/1.0/Eksempel2.xml'
2015-12-01 12:52:28,093 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : DespatchAdvice
2015-12-01 12:52:28,096 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFInvoice/1.6/T14-norsk-profilxy.xml'
2015-12-01 12:52:28,138 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : CreditNote
2015-12-01 12:52:28,141 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFOrder/1.0/Eksempelfil EHF Ordre.xml'
2015-12-01 12:52:28,182 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : Order
2015-12-01 12:52:28,185 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFCatalogue/1.0/Example file EHF Catalogue.xml'
2015-12-01 12:52:28,237 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : Catalogue
2015-12-01 12:52:28,240 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'MessageLevelResponse/1.0/Use Case1.xml'
2015-12-01 12:52:28,276 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : ApplicationResponse
2015-12-01 12:52:28,278 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFInvoice/2.0/T10-B2C.xml'
2015-12-01 12:52:28,317 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : Invoice
2015-12-01 12:52:28,320 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFInvoice/1.6/T14-utland-profil05.xml'
2015-12-01 12:52:28,363 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : CreditNote
2015-12-01 12:52:28,368 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
Identifying 'EHFInvoice/1.6/T14-norsk-profilxx.xml'
2015-12-01 12:52:28,409 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : CreditNote
2015-12-01 12:52:28,412 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
2015-12-01 12:52:28,454 WARN [eu.peppol.outbound.transmission.TransmissionRequestBuilder] Endpoint was set by caller not retrieved from SMP, make sure this is intended be
haviour.
2015-12-01 12:52:28,535 DEBUG [eu.peppol.document.PlainUBLHeaderParser] Creating DocumentParser for type : Invoice
2015-12-01 12:52:28,545 DEBUG [oxalis-com] Constructed SMP url: http://B-770c6f5843e9e302de47ae4026307076.iso6523-actorid-upis.smk.peppolcentral.org/iso6523-actorid-upis%
3A%3A9908%3A991825827/services/busdox-docid-qns%3A%3Aurn%3Aoasis%3Anames%3Aspecification%3Aubl%3Aschema%3Axsd%3AInvoice-2%3A%3AInvoice%23%23urn%3Awww.cenbii.eu%3Atransact
ion%3Abiitrns010%3Aver2.0%3Aextended%3Aurn%3Awww.peppol.eu%3Abis%3Apeppol5a%3Aver2.0%3Aextended%3Aurn%3Awww.difi.no%3Aehf%3Afaktura%3Aver2.0%3A%3A2.1
Tests run: 25, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 7.891 sec <<< FAILURE!
xmlWithNoSBDH(eu.peppol.outbound.transmission.TransmissionRequestBuilderTest)  Time elapsed: 3.049 sec  <<< FAILURE!
java.lang.RuntimeException: Problem with SMP lookup for participant 9908:991825827 and document type urn:oasis:names:specification:ubl:schema:xsd:Invoice-2::Invoice##urn:
www.cenbii.eu:transaction:biitrns010:ver2.0:extended:urn:www.peppol.eu:bis:peppol5a:ver2.0:extended:urn:www.difi.no:ehf:faktura:ver2.0::2.1
        at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:178)
        at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
        at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
        at java.net.Socket.connect(Socket.java:579)
        at java.net.Socket.connect(Socket.java:528)
        at sun.net.NetworkClient.doConnect(NetworkClient.java:180)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:432)
        at sun.net.www.http.HttpClient.openServer(HttpClient.java:527)
        at sun.net.www.http.HttpClient.<init>(HttpClient.java:211)
        at sun.net.www.http.HttpClient.New(HttpClient.java:308)
        at sun.net.www.http.HttpClient.New(HttpClient.java:326)
        at sun.net.www.protocol.http.HttpURLConnection.getNewHttpClient(HttpURLConnection.java:997)
        at sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:933)
        at sun.net.www.protocol.http.HttpURLConnection.connect(HttpURLConnection.java:851)
        at eu.peppol.smp.SmpContentRetrieverImpl.getUrlContent(SmpContentRetrieverImpl.java:52)
        at eu.peppol.smp.SmpLookupManagerImpl.fetchContentsOfSmpUrl(SmpLookupManagerImpl.java:412)
        at eu.peppol.smp.SmpLookupManagerImpl.getServiceMetaData(SmpLookupManagerImpl.java:294)
        at eu.peppol.smp.SmpLookupManagerImpl.getEndpointType(SmpLookupManagerImpl.java:432)
        at eu.peppol.smp.SmpLookupManagerImpl.getEndpointTransmissionData(SmpLookupManagerImpl.java:163)
        at eu.peppol.outbound.transmission.TransmissionTestModule$2.getEndpointTransmissionData(TransmissionTestModule.java:100)
        at eu.peppol.outbound.transmission.TransmissionRequestBuilder.build(TransmissionRequestBuilder.java:168)
        at eu.peppol.outbound.transmission.TransmissionRequestBuilderTest.xmlWithNoSBDH(TransmissionRequestBuilderTest.java:97)


Results :

Failed tests:
  TransmissionRequestBuilderTest.xmlWithNoSBDH:97 ╗ Runtime Problem with SMP loo...

Tests run: 25, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] Oxalis - PEPPOL access point Plug-and-Play ........ SUCCESS [0.810s]
[INFO] Oxalis-api ........................................ SUCCESS [3.870s]
[INFO] Oxalis commons .................................... SUCCESS [10.701s]
[INFO] oxalis-jdbc-dbcp .................................. SUCCESS [1.906s]
[INFO] oxalis-sql ........................................ SUCCESS [2.287s]
[INFO] oxalis-as2 ........................................ SUCCESS [3.861s]
[INFO] Oxalis START inbound (WAR) ........................ SUCCESS [6.498s]
[INFO] Oxalis outbound ................................... FAILURE [13.153s]
[INFO] Oxalis standalone CLI program ..................... SKIPPED
[INFO] oxalis-distribution ............................... SKIPPED
[INFO] oxalis-integration-test ........................... SKIPPED
[INFO] oxalis-jdbc-jndi .................................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 43.415s
[INFO] Finished at: Tue Dec 01 12:52:32 EST 2015
[INFO] Final Memory: 38M/876M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.14.1:test (default-test) on project oxalis-outbound: There are test failures.
[ERROR]
[ERROR] Please refer to C:\oxalis-master\3.1\oxalis-outbound\target\surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :oxalis-outbound

**Comment by klakegg** [comment]
Using wrong SMK/SML doesn't fix anything.

Also, I do not appreciate screaming (unnecessary use of large font).

**Comment by ahmedinweb** [comment]
Hi Klakegg, 
I didn't mean to scream, just copied log from command prompt. My apologies.
which sml host name I should be using? is there any test host name. I am just testing.
mvn clean test.
your response is highly appreciated.

**Comment by ahmedinweb** [comment]
My bad, I was using old code base which is using old SML host.name.
It is fixed with latest code. Thanks.
